test/data_base.cpp

```diff
@@ -1270,7 +1278,7 @@ BOOST_AUTO_TEST_CASE(data_base__invalidate__validate__success)
 
     BOOST_REQUIRE_EQUAL(header_result.metadata.error, error::success);
     BOOST_REQUIRE(header_result.metadata.validated);
-    BOOST_REQUIRE((instance.blocks().get(header_result.hash()).state() & block_state::valid) != 0);
+    BOOST_REQUIRE_EQUAL((instance.blocks().get(header_result.hash()).state() & block_state::valid), block_state::valid);
```

Comment:
Can shorten lines by using intermediate variables, in this case "state" or "result".